static/templates/electionCycles.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset class="js-subcycle-select toggles">
+  {{#each this}}
```

Comment:
Small thing, but this should have a legend right after fieldset:
`<legend class="label">Time period:</legend>`